testsuite/features/secondary/srv_retracted_patches.feature

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) 2015-2021 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+@scope_retracted_patches
+Feature: The Retracted Patches
+
+  Scenario: Installed retracted package should show icon in the system packages list
+    Given I am authorized as "admin" with password "admin"
+    When I install package "rute-dummy-2.1-1.1.x86_64" on this "sle_minion"
+    When I am on the "Software" page of this "sle_minion"
+    And I follow "Packages"
+    And I follow "List / Remove"
+    And I enter "rute-dummy" as the filtered package name
+    And I click on the filter button until page does contain "rute-dummy" text
+    Then the table row for "rute-dummy-2.1-1.1" should contain "errata-retracted" icon
+    Then I remove package "rute-dummy" from this "sle_minion"
+    And I wait until refresh package list on "sle_minion" is finished
+
+  Scenario: Retracted package should not be available for installation
+    Given I am authorized as "admin" with password "admin"
+    When I am on the "Software" page of this "sle_minion"
+    And I follow "Packages"
+    And I follow "Install"
+    Then I should see a "rute-dummy-2.0-1.2" text
+    And I should not see a "rute-dummy-2.1-1.1" text
+
+  Scenario: Retracted package should not be available for upgrade
+    Given I am authorized as "admin" with password "admin"
+    When I install package "rute-dummy-2.0-1.2.x86_64" on this "sle_minion"
+    And I am on the "Software" page of this "sle_minion"
+    And I follow "Packages"
+    And I follow "Upgrade"
+    And I should not see a "rute-dummy-2.1-1.1" text
+    Then I remove package "rute-dummy" from this "sle_minion"
+    And I wait until refresh package list on "sle_minion" is finished
+
+  Scenario: Retracted patch should not affect any systems
+    Given I am authorized as "admin" with password "admin"
+    When I install package "rute-dummy-2.0-1.2.x86_64" on this "sle_minion"
+    And I follow the left menu "Software > Channel List > All"
```

Comment:
Maybe missing usual `I am on the "Software" page of this "sle_minion"` before this line?

Reply:
Not really. This goes to the _software channel_ detail, not to system detail.